test/e2e/data/templates/cluster-template-md-remediation.yaml

```diff
@@ -1,261 +1,259 @@
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
```

Comment:
do we still need this file?

Reply:
yes, this is what will finally be deployed in an E2E test - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/test/e2e/config/ibmcloud-e2e-powervs.yaml#L43